.gitignore

```diff
@@ -5,4 +5,5 @@ test-lib/
 
 java/
 js/**/ElectroGrammar*
-python3/**/ElectroGrammar*
\ No newline at end of file
+python3/**/ElectroGrammar*
+db/*.json
```

Comment:
This is another minor thing but could you configure your editor to add new lines at the end of files?

Reply:
I do have it configured like that. I think I may have used `echo 'db/*.json'>>.gitignore`